app/models/user.rb

```diff
@@ -0,0 +1,16 @@
+class User < ApplicationRecord
+  devise :database_authenticatable, :registerable,
+         :recoverable, :validatable
+
+  enum gender: %i[masculino femenino]
```

Comment:
Change these genders to English. Let the frontend handle how to show those

Reply:
Also, prefer the use of the hash syntax for enums (https://github.com/rootstrap/tech-guides/blob/master/ruby/rails.md#macro-style-methods)